Docker.DotNet.X509/RSAUtil.cs

```diff
@@ -19,20 +15,6 @@ public static X509Certificate2 GetCertFromPFX(string pfxFilePath, string passwor
             return new X509Certificate2(pfxFilePath, password);
         }
 
-#if (NET45 || NET46)
-        public static X509Certificate2 GetCertFromPFXSecure(string pfxFilePath, SecureString password)
```

Comment:
Btw, `X509Certificate2` is supported on `netstandard1.1`+ (which has `net45`in it). However, `SecureString` is `netstandard1.3`(which includes `net46`) so if staying with `netstandard1.3` we keep this just fine.